src/config/Blocks/ListingOptions/utils.js

```diff
@@ -49,6 +49,31 @@ const messages = defineMessages({
     id: 'Path filter filtro',
     defaultMessage: 'Filtro',
   },
+  service_category_link: {
+    id: 'service-category-link',
+    defaultMessage: 'service-category-link',
+  },
+  topic_element: {
+    id: 'topic-element',
+    defaultMessage: 'topic-element',
+  },
+  service_link: {
+    id: 'service-link',
+    defaultMessage: 'service-link',
+  },
+  administration_element: {
+    id: 'administration-element',
+    defaultMessage: 'administration-element',
+  },
```

Comment:
Penso che non serva che questi id siano traducibili in altre lingue, giusto? In tal caso semplificherei togliendo questi e togliendo sotto i vari `intl.formatmessage()` corrispondenti